modules/040-node-manager/images/early-oom/src/psi-monitor.go

```diff
@@ -55,6 +86,25 @@ func main() {
 	}
 }
 
+func probePSISupport() error {
+	fs, err := procfs.NewDefaultFS()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
you have error handling but fail with log.Fatal. Is it necessarily?

Reply:
Not necessary. Bad copypasta while refactoring.